samples/csharp_dotnetcore/14.nlp-with-orchestrator/README.md

```diff
@@ -24,7 +24,7 @@ This bot uses Orchestrator to route user utterances to multiple LUIS models and
 This sample **requires** prerequisites in order to run.
 - Install latest supported version of [Visual C++ Redistributable](https://support.microsoft.com/en-gb/help/2977003/the-latest-supported-visual-c-downloads)
 - Install latest [Bot Framework Emulator](https://github.com/microsoft/BotFramework-Emulator/releases)
-- [.NET Core SDK](https://aka.ms/dotnet-core-applaunch?framework=Microsoft.AspNetCore.App&framework_version=3.1.0&arch=x64&rid=win10-x64) version 3.1
+- [.NET SDK](https://aka.ms/dotnet-core-applaunch?framework=Microsoft.AspNetCore.App&framework_version=3.1.0&arch=x64&rid=win10-x64) version 6.0
```

Comment:
Check this link, you can change it for the ones in the other readmes.

Reply:
Also validate the link its redirecting to the right version
![image](https://user-images.githubusercontent.com/62260472/168332008-728e8cca-347b-483a-8ee1-4eb1490b8726.png)